Jenkinsfile

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'smart-answers'
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  try {
+    stage('Checkout') {
+      checkout scm
+      govuk.cleanupGit()
+      govuk.mergeMasterBranch()
+      govuk.contentSchemaDependency()
```

Comment:
@cbaines There is another ticket to run the smart-answers regression tests. I'll add this task to that ticket. Thanks!